bin/install-theos

```diff
@@ -0,0 +1,598 @@
+			if [[ "${stoolchain,,}" == "y" || "${stoolchain,,}" == "yes" ]]; then
+				read -p "You chose to install the Swift-compatible toolchain. Is this correct? [y/n]" confirm
+				if [[ "${confirm,,}" == "y" || "${confirm,,}" == "yes" ]]; then
+					sudo pkg install -y zstd
+					curl -LO https://github.com/CRKatri/llvm-project/releases/download/swift-5.3.2-RELEASE/swift-5.3.2-RELEASE-ubuntu20.04.tar.zst
+					TMP=$(mktemp -d)
+					tar -xvf swift-5.3.2-RELEASE-ubuntu20.04.tar.zst -C $TMP
+					mkdir -p $THEOS/toolchain/linux/iphone $THEOS/toolchain/swift
+					mv $TMP/swift-5.3.2-RELEASE-ubuntu20.04/* $THEOS/toolchain/linux/iphone/
+					ln -s $THEOS/toolchain/linux/iphone $THEOS/toolchain/swift
+					rm -r swift-5.3.2-RELEASE-ubuntu20.04.tar.zst $TMP
+
+					# Confirm that toolchain is usable
+					if [[ -x "$(command -v otool)" ]]; then
+						update "Successfully installed the Swift-compatible toolchain!"
+					else
+						error "Something appears to have gone wrong -- the toolchain is not accessible. Please try again."
+						exit 8
+					fi
+					break
+				fi
+			elif [[ "${stoolchain,,}" == "n" || "${stoolchain,,}" == "no" ]]; then
+				read -p "You chose to install the non-Swift-compatible toolchain. Is this correct? [y/n]" confirm
+				if [[ "${confirm,,}" == "y" || "${confirm,,}" == "yes" ]]; then
+					curl -LO https://github.com/sbingner/llvm-project/releases/latest/download/linux-ios-arm64e-clang-toolchain.tar.lzma
+					TMP=$(mktemp -d)
+					tar -xvf linux-ios-arm64e-clang-toolchain.tar.lzma -C $TMP
+					mkdir -p $THEOS/toolchain/linux/iphone
+					mv $TMP/ios-arm64e-clang-toolchain/* $THEOS/toolchain/linux/iphone/
+					rm -r linux-ios-arm64e-clang-toolchain.tar.lzma $TMP
+
+					# Confirm that toolchain is usable
+					if [[ -x "$(command -v otool)" ]]; then
+						update "Successfully installed the non-Swift-compatible toolchain!"
+					else
+						error "Something appears to have gone wrong -- the toolchain is not accessible. Please try again."
+						exit 8
+					fi
+					break
+				fi
+			else
+				common "Please pick your desired toolchain to continue."
+			fi
+		done
+	fi
+
+	# Get patched sdks
+	update "Checking for patched sdks..."
+	if [[ -d "$THEOS/sdks/" && "$(ls -A "$THEOS/sdks/" | grep sdk)" ]]; then
+		update "SDKs appear to already be installed."
+	else
+		update "SDKS do not appear to be installed. Installing now..."
+		curl -LO https://github.com/theos/sdks/archive/master.zip
+		TMP=$(mktemp -d)
+		unzip master.zip -d $TMP
+		mv $TMP/sdks-master/*.sdk $THEOS/sdks
+		rm -r master.zip $TMP
+		if [[ -d "$THEOS/sdks/" && "$(ls -A "$THEOS/sdks/" | grep sdk)" ]]; then
+			update "SDKs successfully installed!"
+		else
+			error "Something appears to have gone wrong. Please try again."
+			exit 9
+		fi
+	fi
+}
+
+
+# Determine platform and start work
+if [[ "$PLATFORM" == "Darwin" ]]; then
+	special "Starting install..."
+	if [[ "$(uname -m)" =~ "iP" ]]; then
```

Comment:
Could we change this to test based on `xcode-select` existence? That’s how Theos tests for macOS.

Reply:
Went with xcode-select.